### PR TITLE
Update commands in CONTRIBUTING.md to use new loading options

### PR DESCRIPTION
**Pull request by lparsons**
## Summary Change Description

Update commands in CONTRIBUTING.md to use new loading options.

## Review Notes

Use `load_tissues` command for consistency, instead of `load_study` command.
Update the flags for `load_accucor_msruns

## Checklist

This pull request will be merged once the following requirements are met.  The
author and/or reviewers should uncheck any unmet requirements:

- Review requirements
  - Minimum approvals: 1 
  - No changes requested
  - All blocking issues resolved by reviewers
  - Specific reviewers: @__add_username_here__
    
  - Review period: 2 days 
- Associated issue/pull request requirements:
  
  - [x] All requirements in affected issues marked "resolved" are satisfied
  - [x] All required pull requests are merged *(or none)*
- Basic requirements
  
  - [x] [All linters pass](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#linting)
  - [x] [All tests pass](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#quality-control)
  - [x] All conflicts resolved
- Overhead requirements
  
  - [x] [New/changed method tests implemented/updated *(or no change)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#test-implementation)
  - [ ] [Updated *Unreleased* section of `changelog.md` *(or no change)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/changelog.md)
  - [x] [Migrations created & committed *(or no change)*](https://github.com/Princeton-LSI-ResearchComputing/tracebase/blob/main/CONTRIBUTING.md#migration-process)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -115,9 +115,9 @@ Set up the project's postgres database:
 
     python manage.py loaddata data_types data_formats lc_methods
     python manage.py load_compounds --compounds DataRepo/data/examples/compounds/consolidated_tracebase_compound_list.tsv
-    python manage.py load_study DataRepo/data/examples/tissues/loading.yaml
+    python manage.py load_tissues --tissues DataRepo/data/examples/tissues/loading.yaml
```
Comment (hepcat72) [inline CONTRIBUTING.md:118]: B. This isn't right.  You either need to change `loading.yaml` to whetever the tissue data file is or change `load_tissues` back to `load_study`.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -115,9 +115,9 @@ Set up the project's postgres database:
...
     python manage.py load_animals_and_samples --sample-table-filename DataRepo/data/examples/obob_fasted/obob_samples_table.tsv --animal-table-filename DataRepo/data/examples/obob_fasted/obob_animals_table.tsv --table-headers DataRepo/data/examples/obob_fasted/sample_and_animal_tables_headers.yaml
-    python manage.py load_accucor_msruns --protocol Default --accucor-file DataRepo/data/examples/obob_fasted/obob_maven_6eaas_inf.xlsx --date 2021-04-29 --researcher "Anon" --new-researcher --sample-table-filename DataRepo/data/examples/obob_fasted/obob_samples_table.tsv
+    python manage.py load_accucor_msruns --lc-protocol-name "unknown" --instrument "unknown" --polarity "unknown" --accucor-file DataRepo/data/examples/obob_fasted/obob_maven_6eaas_inf.xlsx --date 2021-04-29 --researcher "Anon" --new-researcher
```
Comment (hepcat72) [inline CONTRIBUTING.md:120]: C. Good catch!  I missed this one.

**Review by hepcat72 — CHANGES_REQUESTED** [review]
Simple mistake, but otherwise, good catch.

File: CONTRIBUTING.md (hunk 1 above)
Comment (lparsons) [reply CONTRIBUTING.md]: Oops, my bad. I actually had fixed it, but didn't commit that last change. Updated now.

**Review by hepcat72 — APPROVED** [review]
🥇

**Pull request merged**